tests/AllTests.php

```diff
@@ -11,24 +11,12 @@
 
 use PHP_CodeSniffer\Util\Tokens;
 
-if (defined('PHP_CODESNIFFER_IN_TESTS') === false) {
-    define('PHP_CODESNIFFER_IN_TESTS', true);
-}
-
-if (defined('PHP_CODESNIFFER_CBF') === false) {
-    define('PHP_CODESNIFFER_CBF', false);
-}
-
-if (defined('PHP_CODESNIFFER_VERBOSITY') === false) {
-    define('PHP_CODESNIFFER_VERBOSITY', 0);
-}
+require_once __DIR__.'/bootstrap.php';
```

Comment:
And if you execute the tests really old-skool PEAR style, it is still needed IIRC. It's fine to keep it the same as earlier. I didn't want to refactor the whole test-suite only for that change which is safe IMHO.